src/models/StakeRequest.ts

```diff
@@ -0,0 +1,79 @@
+import { Address, Bytes32 } from '../types';
+
+export default class StakeRequest {
+  private _id: string;
+
+  private _amount: string;
+
+  private _beneficiary: Address;
+
+  private _gasPrice: string;
+
+  private _gasLimit: string;
+
+  private _nonce: string;
```

Comment:
Curious to know why did we append _ to the names of all private variables ?

Reply:
TS compiler will give an error because of duplicate declaration one with getter and other with the variable declaration. Adding `_` in the parameter to variable resolves it.
```
public get id(): string {
    return this._id;
  }
```
and 
```
private id: string;
```